packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -315,6 +314,12 @@ ensureIndex(Posts,
   }
 );
 
+Posts.addView("tagRelevance", terms => ({
+  options: {
```

Comment:
This needs a cutoff, which should be a selector. As written, if there aren't enough tagged posts, it will fill the remainder with irrelevant stuff.

Reply:
Isn't that covered in the default view? (this does not appear to be an issue for other tag pages with fewer tags, after I fixed the other issue you pointed out)

Reply:
Oh, I see; `TagPage` is using `filterSettings` to set the required tag.